metron-interface/metron-rest/src/test/java/org/apache/metron/rest/controller/KafkaControllerIntegrationTest.java

```diff
@@ -61,6 +62,7 @@
   private static final int KAFKA_RETRY = 10;
   @Autowired
   private KafkaComponent kafkaWithZKComponent;
+  private ComponentRunner runner;
```

Comment:
no, I think we've done enough.  We, honestly, need a test guidance document similar to the parser testing document that I created as part of this ticket.  It's become evident from this exercise that we can provide some over-arching guidance here to limit the probability of us arriving in this situation again.  But that's a follow-on, IMO.